platform/iphone/export/export.cpp

```diff
@@ -300,61 +298,95 @@ void EditorExportPlatformIOS::get_export_options(List<ExportOption> *r_options)
 	}
 }
 
+void EditorExportPlatformIOS::_append_to_file(const Ref<EditorExportPreset> &p_preset, Vector<uint8_t> &pfile, const IOSConfigData &p_config, bool p_debug) {
+	String str;
+	String strnew;
+	str.parse_utf8((const char *)pfile.ptr(), pfile.size());
+
+	String binary_path = p_config.binary_name + "/" + p_config.binary_name;
+
+	// Add reference to our generated Info.plist file
+	strnew += "\nINFOPLIST_FILE = " + binary_path + "-Info.plist;\n";
+
+	// Set the bundle identifier and team from export options
+	String bundle_identifier = p_preset->get("application/identifier");
+	strnew += "PRODUCT_BUNDLE_IDENTIFIER = " + bundle_identifier + ";\n";
+	strnew += "DEVELOPMENT_TEAM = " + (String)p_preset->get("application/app_store_team_id") + ";\n";
+
+	// Pass build settings from Godot export options
+	strnew += "ARCHS = " + p_config.architectures + ";\n";
+	strnew += "FRAMEWORK_SEARCH_PATHS = $(inherited) " + p_config.binary_name + ";\n";
+
+	// Pass linker flags
+	strnew += "OTHER_LDFLAGS = $(inherited)";
+	if (p_config.linker_flags.length() != 0) {
+		strnew += " " + p_config.linker_flags;
+	}
+	strnew += ";\n";
+
+	// if enabled, add development (sandbox) APNS entitlements
+	if ((bool)p_preset->get("capabilities/push_notifications")) {
+		strnew += "\n\"CODE_SIGN_ENTITLEMENTS[sdk=iphoneos*]\" = " + binary_path + ".entitlements;\n";
```

Comment:
We can probably drop this in the future and let Xcode generate entitlements as well what it's already doing in this PR.

Reply:
I'm a fan of letting XCode do as much of the work for us as possible, that way if apple decides to change/enhance it we should end up piggy backing on that change.